ibctest/ibc_test.go

```diff
@@ -91,7 +105,7 @@ func (r *relayer) log() *zap.Logger {
 	return zaptest.NewLogger(r.t)
 }
 
-func (r *relayer) AddChainConfiguration(ctx context.Context, chainConfig ibc.ChainConfig, keyName, rpcAddr, grpcAddr string) error {
+func (r *relayer) AddChainConfiguration(ctx context.Context, _ ibc.RelayerExecReporter, chainConfig ibc.ChainConfig, keyName, rpcAddr, grpcAddr string) error {
```

Comment:
Can't say I've seen the blank identifier used in a function's parameters like this; pretty neat way to satisfy an interface.